runhouse/rns/function.py

```diff
@@ -392,14 +430,14 @@ def remote(self, *args, **kwargs):
             # TODO print this nicely
             logger.info(
                 f"Submitted remote call to cluster. Result or logs can be retrieved"
-                f'\n with run_key "{run_key}", e.g. '
-                f'\n`{cluster_name}.get("{run_key}", stream_logs=True)` in python '
-                f'\n`runhouse logs "{self.system.name}" {run_key}` from the command line.'
+                f'\n with run name "{run_name}", e.g. '
+                f'\n`{cluster_name}.get("{run_name}", stream_logs=True)` in python '
```

Comment:
this reads kinda clunky and outputs a large chunk of text, we should have this somewhere in docs instead of listing out 4 separate examples of using the run_name